install_config/aggregate_logging.adoc

```diff
@@ -1343,6 +1343,206 @@ This causes Fluentd to write logs to the Fluentd default location, *_/var/log/fl
 oc set env ds/fluentd LOGGING_FILE_PATH=console
 ----
 
+[[fluentd-merge-json]]
+*Configuring log collection JSON parsing*
+
+You can configure the Fluentd log collector to determine if a log message is in *JSON* format and merge
+the message into the JSON payload document posted to Elasticsearch. This feature is disabled by default.
+
+You can enable or disable this feature by editing the `MERGE_JSON_LOG` environment variable in the *fluentd* daemonset.
```

Comment:
You should also change your ansible inventory/vars file to make sure this set correctly when you redeploy:
```yaml
openshift_logging_fluentd_merge_json_log: "false"
```
This will ensure that Fluentd will keep the `MERGE_JSON_LOG=false` setting upon re-running ansible.